rpc_jobs/standard_job_postmerge.yml

```diff
@@ -2,14 +2,36 @@
     name:       "gating-post-merge"
     repo_name:  "rpc-gating"
     repo_url:   "https://github.com/rcbops/rpc-gating"
-    # `branch` defaults to `master` and can be omitted.
-    #branch:     "master"
+    branch:     "master"
     image:      "xenial"
     scenario:   "functional"
     action:     "test"
     jira_project_key: "RE"
+    trigger_build: "PM_{repo_name}-{branch}-{image}-{scenario}-{action}"
     jobs:
       - 'PM_{repo_name}-{branch}-{image}-{scenario}-{action}'
+      - 'PM-push-trigger_{repo_name}-{branch}-{image}-{scenario}-{action}'
+
+# This job is specified as `freestyle` as using a pipeline job with github
+# trigger causes the job to be triggered irrespective of which branch was
+# pushed to.
+- job-template:
+    name: "PM-push-trigger_{repo_name}-{branch}-{image}-{scenario}-{action}"
+    project-type: freestyle
+    scm:
+      - git:
+          url: "{repo_url}"
+          branches:
+           - "{branch}"
```

Comment:
As with the standard PM job, can the default ``branch`` value be set to 'master' so that the setting can be omitted in the project (to simplify the usage of the template).

Reply:
The problem is that `branch` can't be omitted from the project when it's referenced in the `trigger_build` variable.

Reply:
Ah, best to note that in the docs then.

Reply:
Added [note to docs](https://rpc-openstack.atlassian.net/wiki/pages/diffpagesbyversion.action?pageId=19005457&selectedPageVersions=49&selectedPageVersions=48).